zzz_generated/lighting-app/zap-generated/endpoint_config.h

```diff
@@ -981,128 +983,393 @@
+/*   incoming_client */ 0x00000001 /* AttestationResponse */ , 0x00000003 /* CertificateChainResponse */ , 0x00000005 /* OpCSRResponse */ , 0x00000008 /* NOCResponse */ , chip::kInvalidCommandId /* end of list */, \
+/* Endpoint: 0, Cluster: Fixed Label (server) */\
+/* Endpoint: 0, Cluster: User Label (server) */\
+/* Endpoint: 1, Cluster: Identify (server) */\
+/*   incoming_server */ 0x00000000 /* Identify */ , 0x00000001 /* IdentifyQuery */ , 0x00000040 /* TriggerEffect */ , chip::kInvalidCommandId /* end of list */, \
+/*   incoming_client */ 0x00000000 /* IdentifyQueryResponse */ , chip::kInvalidCommandId /* end of list */, \
+/* Endpoint: 1, Cluster: Groups (server) */\
+/*   incoming_server */ 0x00000000 /* AddGroup */ , 0x00000001 /* ViewGroup */ , 0x00000002 /* GetGroupMembership */ , 0x00000003 /* RemoveGroup */ , 0x00000004 /* RemoveAllGroups */ , 0x00000005 /* AddGroupIfIdentifying */ , chip::kInvalidCommandId /* end of list */, \
+/*   incoming_client */ 0x00000000 /* AddGroupResponse */ , 0x00000001 /* ViewGroupResponse */ , 0x00000002 /* GetGroupMembershipResponse */ , 0x00000003 /* RemoveGroupResponse */ , chip::kInvalidCommandId /* end of list */, \
+/* Endpoint: 1, Cluster: On/Off (server) */\
+/*   incoming_server */ 0x00000000 /* Off */ , 0x00000001 /* On */ , 0x00000002 /* Toggle */ , 0x00000040 /* OffWithEffect */ , 0x00000041 /* OnWithRecallGlobalScene */ , 0x00000042 /* OnWithTimedOff */ , chip::kInvalidCommandId /* end of list */, \
+/* Endpoint: 1, Cluster: Level Control (server) */\
+/*   incoming_server */ 0x00000000 /* MoveToLevel */ , 0x00000001 /* Move */ , 0x00000002 /* Step */ , 0x00000003 /* Stop */ , 0x00000004 /* MoveToLevelWithOnOff */ , 0x00000005 /* MoveWithOnOff */ , 0x00000006 /* StepWithOnOff */ , 0x00000007 /* StopWithOnOff */ , chip::kInvalidCommandId /* end of list */, \
+/* Endpoint: 1, Cluster: Descriptor (server) */\
+/* Endpoint: 1, Cluster: Color Control (server) */\
+/*   incoming_server */ 0x00000000 /* MoveToHue */ , 0x00000001 /* MoveHue */ , 0x00000002 /* StepHue */ , 0x00000003 /* MoveToSaturation */ , 0x00000004 /* MoveSaturation */ , 0x00000005 /* StepSaturation */ , 0x00000006 /* MoveToHueAndSaturation */ , 0x00000007 /* MoveToColor */ , 0x00000008 /* MoveColor */ , 0x00000009 /* StepColor */ , 0x0000000A /* MoveToColorTemperature */ , 0x00000040 /* EnhancedMoveToHue */ , 0x00000041 /* EnhancedMoveHue */ , 0x00000042 /* EnhancedStepHue */ , 0x00000043 /* EnhancedMoveToHueAndSaturation */ , 0x00000044 /* ColorLoopSet */ , 0x00000047 /* StopMoveStep */ , 0x0000004B /* MoveColorTemperature */ , 0x0000004C /* StepColorTemperature */ , chip::kInvalidCommandId /* end of list */, \
+/* Endpoint: 1, Cluster: Occupancy Sensing (server) */\
+/* Endpoint: 2, Cluster: On/Off (client) */\
```

Comment:
One more thing: this part does not look right to me.... Why are we generating things for client sides of clusters here?

Let's leave this as-is for now, but do a followup to not generate this... and probably not generate ClusterRevision for client clusters either, since it's not observable...

Reply:
I think the "client" cluster should not appear on `endpoint_config.h` given the fact that client has no attribute storage and actual command dispatching routine.

Reply:
We'd have to check all the places that use that metadata in the code (e.g. looking up which client clusters are on which endpoint)....  Out of scope for this PR, like I said, but please make sure a followup is filed.

Reply:
Created a ticket https://github.com/project-chip/connectedhomeip/issues/14508